### PR TITLE
Partially rename to getmesh for internal usage.

### PR DESCRIPTION
**Pull request by mathetake**
Signed-off-by: Takeshi Yoneda <takeshi@tetrate.io>

### CODE REVIEW

**Comment by mathetake** [comment]
@deva26 need to rename cloudsmith project to getmesh accordingly to pass tests.

File: cmd/switch_test.go [hunk 1]
```diff
@@ -75,11 +74,11 @@ func Test_switchParse(t *testing.T) {
 		FlavorVersion: 0,
 	}
 
-	require.NoError(t, getistio.SetIstioVersion(home, d))
+	require.NoError(t, getmesh.SetIstioVersion(home, d))
 	require.NoError(t,
-		os.MkdirAll(strings.TrimSuffix(istioctl.GetIstioctlPath(home, d), "/istioctl"), 0755))
+		os.MkdirAll(strings.TrimSuffix(istioctl.getmeshctlPath(home, d), "/istioctl"), 0755))
```
Comment (deva26) [inline cmd/switch_test.go:79]: I think we may to retain this as it is related to Istioctl. WDYT?

**Comment by deva26** [comment]
@mathetake - There are some external dependencies like the arichive repos and download URL, etc. Till they are changed, we need to keep the old values. I think it would be better to address them in a different PR.

**For now, let us proceed with the following assumption:**
github repo name would remain the same getisio, getistio.io
URL to download the CLI  would be the same (https://dl.getistio.io/public/raw/files/download.sh)

**Below are internal - so we can getistio name as is**
Repo to pull Istio Archives from
Repo where the cli binaries are saved for distribution

**Comment by mathetake** [comment]
OK for now I think it's ok to use getistio partially (https://github.com/tetratelabs/getistio/pull/40/commits/2ccd8116a12084833cb562b27e9a3314c5ecb13c) but in anyway we should remove all the usage of "getistio" no matter it is used internally or not.. this is just causing a mess in the code base.

**Comment by deva26** [comment]
> OK for now I think it's ok to use getistio partially ([2ccd811](https://github.com/tetratelabs/getistio/commit/2ccd8116a12084833cb562b27e9a3314c5ecb13c)) but in anyway we should remove all the usage of "getistio" no matter it is used internally or not.. this is just causing a mess in the code base.

Agree. We shall do that in phases. FIrst phase is the existing changes you have done, second changing archive repo names, URLs, etc, third github repo names.

**Comment by mathetake** [comment]
yeah, so I think we are ready to go for the current change now.

File: CODEOWNERS [hunk 2]
```diff
@@ -1 +1 @@
-* @tetratelabs/getistio-team
+* @tetratelabs/getmesh-team
```
Comment (mathetake) [inline CODEOWNERS:1]: @deva26 we should have a team wit getistio or whatever the new name 😄

Comment (mathetake) [reply CODEOWNERS:1]: let's just remove for now then since this codeowner file is not working as we expected.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# GetIstio [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+# getmesh [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-![Simplify Istio with GetIsito!](getIstio.png)
+![Simplify Istio with GetIsito!](getmesh.png)
```
Comment (mathetake) [inline README.md:3]: What about logo?

Comment (deva26) [reply README.md:3]: Logo remains the same.

**Review by deva26 — COMMENTED** [review]
Thanks, @mathetake

**Review by deva26 — APPROVED** [review]

**Pull request merged**